src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -94,6 +88,49 @@ CAC.Map.Control = (function ($, Handlebars, cartodb, L, turf, _) {
         initializeLayerControl();
     }
 
+    /**
+     * Helper to determine if Leaflet base map has already loaded tiles.
+     *
+     * @returns {boolean} True if base map tiles have already been loaded
+     */
+    function isLoaded() {
+        return loaded;
+    }
+
+    /**
+     * Load base map tiles and set map on associated controls.
+     *
+     * @param {Object} ctl Reference to this control (useful if calling from elsewhere)
+     */
+    function loadMap(ctl) {
```

Comment:
Could consider just calling this method using `apply()` instead of explicitly requiring the `ctl` argument. 

It feels a bit odd to have a prototype method require a reference to itself as the first argument. But that's as much of a way these files are setup as anything.